src/code/z_actor.c

```diff
@@ -865,10 +865,10 @@ void Actor_MoveForward(Actor* actor) {
 }
 
 void func_8002D908(Actor* actor) {
-    f32 sp24 = Math_CosS(actor->world.rot.x) * actor->speedXZ;
+    f32 sp24 = Math_CosS(actor->world.rot.x) * actor->speed;
```

Comment:
`sp24` can now be `speedXZ` without looking weird 👀

Reply:
yea this is changed in #1476, tho i guess we can do it now